hazenlib/tasks/acr_uniformity.py

```diff
@@ -77,16 +144,16 @@ def get_integral_uniformity(self, dcm):
         """
         img = dcm.pixel_array
         # Required pixel radius to produce ~200cm2 ROI
-        r_large = np.ceil(80 / self.ACR_obj.dx).astype(int)
```

Comment:
Just curious for the change in this value (from 80 -> 75)? Should this be a keyword argument?